src/atoms/tile/AppTile.ts

```diff
@@ -0,0 +1,41 @@
+import { TTile } from '../../types';
+
+export function AppTile({
+    width,
+    height,
+    content,
+    textSize,
+    imgOptions,
+    onClick,
+}: TTile): HTMLElement {
+    const tile = document.createElement('div');
+    tile.className = `rounded-lg bg-tile-bg-gray flex`;
+    tile.classList.add(width, height);
+
+    if (content.startsWith('http')) {
+        const img = document.createElement('img');
+        if (imgOptions) {
+            img.classList.add(...imgOptions);
```

Comment:
Consider adding custom classes after generic classes (line 21) in case you want to override generics

Reply:
There classes are added to img element but in line 21 classes are added to tile element